### PR TITLE
#1773 Project import on projects page: add option to add missing users

### PR DESCRIPTION
**Pull request by UWinch**
Allow administrators to also import permissions and create missing users when importing projects on the projects overview page. 

**What's in the PR**
* Re-use of import form from administration page

**How to test manually**
* Admin user should be able to import project from Projects overview with additional checkboxes for users and user permissions
* Normal user should not see the import button at all
* Project Creator should only be able to import the project without users

### CODE REVIEW

File: inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java [hunk 1]
```diff
@@ -65,23 +52,19 @@
...
 import de.tudarmstadt.ukp.clarin.webanno.model.Project;
 import de.tudarmstadt.ukp.clarin.webanno.model.ProjectPermission;
 import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
 import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
-import de.tudarmstadt.ukp.clarin.webanno.support.ZipUtils;
 import de.tudarmstadt.ukp.clarin.webanno.support.dialog.ConfirmationDialog;
 import de.tudarmstadt.ukp.clarin.webanno.support.lambda.LambdaAjaxLink;
 import de.tudarmstadt.ukp.clarin.webanno.support.lambda.LambdaBehavior;
 import de.tudarmstadt.ukp.clarin.webanno.ui.core.page.ApplicationPageBase;
+import de.tudarmstadt.ukp.clarin.webanno.ui.project.ProjectImportPanel;
```
Comment (reckart) [inline inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java:67]: Amazing :) I didn't remember this was already a re-usable component.

**Comment by reckart** [comment]
Wasn't there some discussion on whether non-admin users should be able to import permissions?

**Comment by reckart** [comment]
> Works for me, when I click checkboxes, then the dropdown closes, that is annoying.

Do you also experience this in the project settings page?

**Comment by jcklie** [comment]
> > Works for me, when I click checkboxes, then the dropdown closes, that is annoying.
> 
> Do you also experience this in the project settings page?

I do not, there it works as expected.

**Comment by reckart** [comment]
I've had that once elsewhere with the buttons behaving weird - eventually I got it working, but I don't remember what the trick was. I think probably something in the HTML *surrounding* the import panel.

File: inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java (hunk 1 above)
Comment (UWinch) [reply inception-ui-core/src/main/java/de/tudarmstadt/ukp/inception/ui/core/dashboard/projectlist/ProjectsOverviewPage.java:67]: Thanks Richard from the past!

**Comment by UWinch** [comment]
> Wasn't there some discussion on whether non-admin users should be able to import permissions?

Yes, but we did not make a decision on this, so I'm keeping it as-is for now. 

I also fixed the problem that the dropdown closed when clicking on it. So, I'm merging this now unless someone wants to test again?

**Pull request merged**